controllers/articles_controller.js

```diff
@@ -12,7 +13,9 @@ exports.getArticleById = async (req, res, next) => {
 
 exports.getArticles = async (req, res, next) => {
     try {
-        const articles = await selectAllArticles()
+        const { topic } = req.query
+        await selectAllTopics(topic)
```

Comment:
Do we need to call this here? I can't see where you're using any return value of it?